extras/indexingExample/src/main/assembly/assembly.xml

```diff
@@ -33,6 +33,7 @@ under the License.
             <includes>
                 <include>org.apache.rya:rya.indexing:*:accumulo-server</include>
                 <include>org.locationtech.geomesa:geomesa-accumulo-distributed-runtime:*</include>
+                <include>mil.nga.giat:geowave-datastore-accumulo:*</include>
```

Comment:
I think the dependency is something like `geowave-deploy-0.9.3-accumulo`?  I can't find this dependency on their maven repository, though